.github/workflows/pr_tests_gpu.yml

```diff
@@ -0,0 +1,218 @@
+name: Fast GPU Tests on PR 
+
+on:
+  pull_request:
+    branches: main
+    paths:
+      - "src/diffusers/models/modeling_utils.py"
+      - "src/diffusers/models/model_loading_utils.py"
+      - "src/diffusers/pipelines/pipeline_utils.py"
+      - "src/diffusers/pipeline_loading_utils.py"
```

Comment:
I think `loaders.py` could also be included to avoid situations like https://github.com/huggingface/diffusers/pull/1085